core/profiles/src/main/java/software/amazon/awssdk/profiles/ProfileFile.java

```diff
@@ -209,6 +264,23 @@ private Map<String, Map<String, Profile>> convertToProfilesSectionsMap(
         return result;
     }
 
+    private ProfileFile registerBuildDetails(Builder.BuildDetails details) {
+        this.buildDetails.add(details);
+        return this;
+    }
+
+    private ProfileFile registerBuildDetails(Collection<Builder.BuildDetails> details) {
+        this.buildDetails.addAll(details);
+        return this;
+    }
```

Comment:
"register" is not really a verb we use in the codebase. You can take a look at our coding guidelines here https://github.com/aws/aws-sdk-java-v2/blob/master/docs/design/ClientConfiguration.md

probably something like `addBuildDetails` and `buildDetails` respectively.

Reply:
Updated method names.